source/CommonJobs/CommonJobs.Mvc.UI/Content/Site.css

```diff
@@ -123,4 +123,7 @@ body div.bootstrap-scope.navbar-container
 .page-header label.checkbox
 {
     margin-right: 10px;
-}
\ No newline at end of file
+}
+
+/*header*/
+.header__main{width:100%}.header__main .header__holder{background-color:#000;position:relative}.header__main .suit-wrapper,.header__main .content__main{*zoom:1;width:auto;max-width:75em;float:none;display:block;margin-right:auto;margin-left:auto;padding-left:0;padding-right:0;position:relative;height:7.1875em;padding-left:20px;padding-right:20px}.header__main .suit-wrapper:before,.header__main .content__main:before,.header__main .suit-wrapper:after,.header__main .content__main:after{content:'';display:table}.header__main .suit-wrapper:after,.header__main .content__main:after{clear:both}.header__main .logo{display:block;max-width:12.5em;float:left;position:relative;top:50%;left:auto;-webkit-transform:translate(0, -50%);-ms-transform:translate(0, -50%);transform:translate(0, -50%)}.header__breadcrum{float:left;margin-left:30px;color:#fff;position:relative;top:50%;left:auto;-webkit-transform:translate(0, -50%);-ms-transform:translate(0, -50%);transform:translate(0, -50%)}.header__sub{float:right;position:relative;top:50%;left:auto;-webkit-transform:translate(0, -50%);-ms-transform:translate(0, -50%);transform:translate(0, -50%)}.header__sub .icon-question-mark{font-size:1.6875em;color:#fff;vertical-align:middle;margin-right:20px}.header__user{font-weight:700;color:#fff}.header__user a{opacity:0.50;color:inherit;-webkit-transition:all 0.25s ease;transition:all 0.25s ease}
```

Comment:
Same question about uncompressed sources and developers being able to work on them.